glide.lock

```diff
@@ -68,6 +68,8 @@ imports:
   version: a3647f8e31d79543b2d0f0ae2fe5c379d72cedc0
 - name: github.com/pelletier/go-toml
   version: 2009e44b6f182e34d8ce081ac2767622937ea3d4
+- name: github.com/philhofer/fwd
```

Comment:
is this used anywhere or just an accidental add-on by glide?

Reply:
It's used by `msgp`

Reply:
👍